base/doc/ltnews38.tex

```diff
@@ -283,29 +409,80 @@ \subsection{In the programming layer}
  >\fp_eval:n{dinner(duck)}
 $
 \ExplSyntaxOff
-\end{document}
 \end{verbatim}
 
+The computation above would then generate
+\begin{quote}
+\ExplSyntaxOn
+\fp_new_variable:n{duck}
+\fp_new_function:n{dinner}
+\fp_set_function:nnn{dinner}{duck}
+                    {duck - 0.25 * duck}
+\fp_set_variable:nn{duck}{1}
```

Comment:
as above

Reply:
again not my call, but it isn't impossible to always take a quarter from what remains (okay you get more hungry over time but ...)

Reply:
But is a “duck” just an (fp) number theses days!!  So sad.